### DISCUSSION TITLE
Error using Webscraper

### DISCUSSION THREAD

**Discussion by Toolfolks** [FlowiseAI/Flowise#1746] [Q&A]
I am testing on this site. 
`https://www.brscc.co.uk/`
I have tried the three different scraper types.
When upserting I get:
TypeError: (intermediate value) is not iterable (cannot read property undefined)

![image](https://github.com/FlowiseAI/Flowise/assets/125031610/a9caea95-496e-493f-99bf-6ca9d9b8ba7e)

**Comment by chungyau97** [comment]
Hi @Toolfolks, we merged this PR #1740 which will solve the issue you're facing 

![image](https://github.com/FlowiseAI/Flowise/assets/33013947/1fa5da21-1e57-4087-b214-e92925a612f6)

**Reply by Toolfolks** [reply]
I have updated to ver 1.51.  I have used the following:
1 Cheerio Web Scraper Additional parameters set to Web Crawl.
2. Managed links and crawled the website and got 100's of link ( too many to count )
3. Saved the results
4. Saved the Chatbot.
5. Upsert and get error No relative links found.
6.  What method do you use to get the results.?
7.  What is the 'correct' procedure to use ?

**Comment by Vortigern-source** [comment]
Why his this been closed?
